resources/crds/hammerdb_crds/ripsaw_v1alpha1_hammerdb_cr.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: hammerdb-benchmark
+  namespace: my-ripsaw
+spec:
+  elasticsearch:
+    url: http://my.elasticsearch.server:80
+    index_name: ripsaw-hammerdb
+  metadata:
+    collection: false
+  workload:
+    name: hammerdb
+    args:
+      #image: quay.io/test/hammerdb:latest # add custom hammerdb image
+<<<<<<< HEAD
```

Comment:
need to fix this merge conflict

Reply:
Thx, fix it.
Please rerun hammerdb again